check_rate: 40  # Hz
constraints:
  - source_topic: "/image_raw"
    warn_delay: 0.2
    error_delay: 1
